packages/homepage/src/components/Navigation/index.js

```diff
@@ -1,4 +1,6 @@
+/* eslint-disable import/no-cycle */
 import React, { useEffect, useState } from 'react';
+import OutsideClickHandler from 'react-outside-click-handler';
```

Comment:
Do we already have this in the `package.json` of `homepage`? 🤔

Reply:
Its lerna, it uses all the same one

Reply:
I know, but it can bite you back when you would have deleted it somewhere else and you don't have it in this `package.json`

Reply:
ohh, we use in another places its fine